README.md

```diff
@@ -1,2 +1,75 @@
-# uuid-msgpack
-Library for integrating github.com/google/uuid with gopkg.in/vmihailenco/msgpack.v2.
+# UUID msgpack
+Library to integrate github.com/google/uuid with gopkg.in/vmihailenco/msgpack
+
+
+Installation
+------------
+	go get github.com/citilinkru/camunda-client-go/v2
+
+Example
+-------
+```go
+package main
+
+import (
+	_ "code.citik.ru/gobase/uuid-msgpack" // Обязательно нужно сделать импорт!
```

Comment:
Please fix for public import github.com...
and translate the Russian comment into English

Reply:
fixed